### ISSUE TITLE
Sonoff RF Bridge toggling?

### ISSUE THREAD

**Issue by sickao3** [skydiver/ewelink-api#71]
I can see my Sonoff RF Bridge listed with all his RF list commands. Is there any way to toggle those ?

**Comment by alessandronocera89** [comment]
hi! I know are passed over 2 years but I've discovered these APIs right now and I've implemented an home-made service that allows me to manipulate with my own API the devices like the SONOFF MINI...and I was wondering to integrate with the RF Bridge also but I've tried to call the api with the channel that I guess was correct (after checks with .getDevice(<the-id>) but the response is: 
`{
    "error": false
}`

I've used the "set-device-power-state" method off course by passing "toggle" but nothing appears. Did you know if it is possible kinda use this api also for RG Bridge? I google a lot and try myself to find the solution but now I'm blocked. 

Thank you and hope in your reply!
